source/server/admin/admin.cc

```diff
@@ -456,15 +467,26 @@ Http::Code AdminImpl::handlerClusters(absl::string_view url,
 }
 
 void AdminImpl::addAllConfigToDump(envoy::admin::v3::ConfigDump& dump,
-                                   const absl::optional<std::string>& mask,
-                                   bool include_eds) const {
+                                   const absl::optional<std::string>& mask, bool include_eds,
+                                   bool include_active_listener_unready_targets,
+                                   bool include_warming_listener_unready_targets) const {
```

Comment:
It's over complicated. The `not ready listener in active list` is an implementation details.
User would only care about the the unready listeners regardless they are in active list or warm list.

Reply:
Will merge them together.